src/idd/ui/scrollable_area.py

```diff
@@ -10,19 +14,73 @@ class TextScrollView(RichLog):
     DEFAULT_CSS = """
     TextScrollView {
         height: 100%;
-        scrollbar-size: 1 1;
+        scrollbar-size: 1 1;  
     }
     """
 
+    # Terminal-specific selection tips
+    SELECTION_TIPS = {
+        "Terminal.app": "Option(⌥)+Click to select text",
+        "iTerm2": "Cmd(⌘)+Shift+C: Copy mode | Option(⌥)+Click: Selection",
+        "Warp": "Shift+Click to select text",
+        "default": "Use terminal's selection mechanism to copy text" #default mac terminal
```

Comment:
```suggestion
        "default": "Use terminal's selection mechanism to copy text (Maybe Shift+Click)" #default mac terminal
```

Does this sound better?